GeorgianMemenator/Scenes/CatTagsList/CatTagsListViewModel.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  CatTagsListViewModel.swift
+//  GeorgianMemenator
+//
+//  Created by Giorgi Makadze on 09.01.2023.
+//  Copyright © 2023 Shalva Avanashvili. All rights reserved.
+//
+
+import SwiftUI
+import Foundation
+
+struct CatTagsListViewModel: View {
```

Comment:
ViewModels should not be child class of `View`, it should conform to `ObservableObject` instead. Please see RedditViewModel